corehq/apps/settings/views.py

```diff
@@ -664,13 +665,14 @@ def column_names(self):
             _("Project"),
             _("IP Allowlist"),
             _("Created"),
+            _("Last Used"),
             _("Status"),
             _("Actions"),
         ]
 
     def _to_user_time(self, value):
         return (ServerTime(value)
-                .user_time(self.request.couch_user.get_time_zone())
+                .user_time(pytz.timezone(self.request.couch_user.get_time_zone()))
```

Comment:
I was not sure if that was the best place to put that or not. Please let me know if there is a better one.

Reply:
Do we need to handle the case where `get_time_zone` returns `None`?

Reply:
@snopoke Should the logging be removed then? Because both the project timezone and the user override are stored as string. So there are a lot of warnings logged.